api/iceberg-service/src/main/java/org/apache/polaris/service/types/NotificationRequest.java

```diff
@@ -22,10 +22,6 @@
 import io.swagger.annotations.ApiModelProperty;
 import java.util.Objects;
 
-@jakarta.annotation.Generated(
```

Comment:
I believe this annotation is improperly placed, but happy to put it back if I'm wrong.